homeassistant/components/enphase_envoy/sensor.py

```diff
@@ -364,6 +364,87 @@ class EnvoyCTSensorEntityDescription(SensorEntityDescription):
     for phase in range(3)
 }
 
+CT_STORAGE_SENSORS = (
+    EnvoyCTSensorEntityDescription(
+        key="lifetime_battery_discharged",
+        translation_key="lifetime_battery_discharged",
+        native_unit_of_measurement=UnitOfEnergy.WATT_HOUR,
+        state_class=SensorStateClass.TOTAL_INCREASING,
+        device_class=SensorDeviceClass.ENERGY,
+        suggested_unit_of_measurement=UnitOfEnergy.MEGA_WATT_HOUR,
+        suggested_display_precision=3,
+        value_fn=lambda ct: ct.energy_delivered,
```

Comment:
This could be replaced with `attrgetter('energy_delivered')`

`from operator import attrgetter`